src/sidebar/index.js

```diff
@@ -217,6 +218,7 @@ function startAngularApp(config) {
     .register('flash', flashService)
     .register('frameSync', frameSyncService)
     .register('groups', groupsService)
+    .register('loadAnnotationsService', loadAnnotationsService)
```

Comment:
I think it would be a good idea to come up with a policy for which of these services get a `Service` suffix and which don't, so we can eventually achieve better naming consistency.

Reply:
From verbal conversation earlier today: we're going to follow up on this in our next front-end sync conversation.